tests/integration/components/task-details-test.js

```diff
@@ -133,3 +133,15 @@ test('it saves', function(assert) {
   page.save.click();
   assert.equal(page.commentBody.text, 'A new body', 'The body is saved');
 });
+test('if the task is null render the no description element', function(assert) {
+  stubService(this, 'mention-fetcher', {
+    prefetchBodyWithMentions() {
+      return null;
+    }
+  });
+
+  this.set('task', mockTask);
+
+  page.render(hbs`{{task-details task=task}}`);
+  assert.ok(page.nullCommentBody.isVisible, 'The message for no comment body is rendered');
```

Comment:
Can we test for the opposite case where there _is_ a body (i.e. this should _not_ render)?